File: Web/Direct4Me.Minimal.Api/Models/FaceUnlock/FaceUnlockResponse.cs
using System.ComponentModel.DataAnnotations;

namespace Direct4Me.Minimal.Api.Models.FaceUnlock;

public record FaceUnlockResponse(
    [Required] string Guid,
    [Required] string Email,
    [Required] string Fullname,
    [Required] int TotalLogins,
    [Required] DateTime LastAccess);

File: Mobile/app/src/main/java/com/um/feri/direct4me/android/api/model/faceUnlock/FaceUnlockResponse.kt
package com.um.feri.direct4me.android.api.model.faceUnlock

import java.util.Date

data class FaceUnlockResponse(
    val guid: String,
    val email: String,
    val fullname: String,
    val totalLogins: Int,
    val lastAccess: Date
)
